sample-app/src/test/java/com/fabernovel/statefullayout/utils/statefulStateMatcher.kt

```diff
@@ -0,0 +1,14 @@
+package com.fabernovel.statefullayout.utils
+
+import android.view.View
+import androidx.annotation.IdRes
+import androidx.test.espresso.matcher.ViewMatchers.withClassName
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withParent
+import com.fabernovel.statefullayout.StatefulLayout
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Matchers.equalTo
+
+fun isStatefulLayoutState(@IdRes stateId: Int): Matcher<View> =
```

Comment:
for a more readable DSL, I think it's better to start the matcher name with a `with` prefix, to have something like that in the end : `onView(withStatefulState(stateId))`...